.github/ISSUE_TEMPLATE/UX_Template.yml

```diff
@@ -0,0 +1,58 @@
+name: UX Request
+description: UX Request Form
+labels: [UX/UI Issue, Graphic design]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a UX request, please search for existing issues.
+
+        ---
+
+  - type: textarea
+    id: UX-description
+    attributes:
+      label: UX Description
+      description: Describe the request
+    validations:
+      required: true
+
+  - type: dropdown
+    id: request-type
+    attributes:
+      label: Request type
+      description: "What type of request is this?"
+      options:
+        - "A logo design"
+        - "An icon"
+        - "An infographic/chart"
+        - "A template or design for printed materials"
+        - "Swag design"
+        - "Graphic design not covered by the above"
+
+  - type: dropdown
+    id: user-experience
+    attributes:
+      label: User Experience Request type
```

Comment:
Seems a little repetitive ('request type' then 'user experience request type'), and I think they'll both appear but the options seem kind of orthogonal, e.g. I could do 'request type: logo design' and 'user experience request type: website mockup'? I'm wondering if these should just be a single dropdown.